pkg/adapter/BUILD

```diff
@@ -28,6 +28,8 @@ go_test(
     name = "small_tests",
     size = "small",
     srcs = [
+        "applicationLogs_test.go",
```

Comment:
aside: this reminds me that we probably should have all lowercase filenames (so, `applicationlogs.go` and `applicationlogs_test.go`, here).